.eslintrc.json

```diff
@@ -0,0 +1,23 @@
+{
+  "extends": [
+    "airbnb",
+    "prettier",
+    "plugin:prettier/recommended"
+  ],
+  "plugins": ["prettier"],
+  "root": true,
+  "env": {
+    "es6": true,
+    "browser": true
+  },
+  "globals": {
+    "expect": true,
+    "it": true,
+    "describe": true,
```

Comment:
Instead of adding expect/it/describe as globals, we could instead add jest to env: `"env": { "jest": true }`